scripts/generate_certs.py

```diff
@@ -0,0 +1,156 @@
+
+You must run this script from an Mbed TLS root.
+
+"""
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import os
+import re
+import subprocess
+import tempfile
+
+class UpdaterCerts:
+    """ Updater for certs.c"""
+
+    def __init__(self):
+        """Instantiate the updater for certs.c
+
+        certs: path to certs.c
+        """
+        self.certs = "tests/src/certs.c"
+
+    @staticmethod
+    def type_var_bin(name, filename):
+        """ Generates code snipper to be added
+        name: name of variable
+        filename: where value of variable stored
+        """
+        output = "const unsigned char " + name + "[] = {\n"
+        data = subprocess.check_output(["xxd", "-i", filename]).decode("utf-8")
+        output += '   ' + '   '.join(data.splitlines(keepends=True)[1:-2])
+        output += "};\n"
+        return output
+
+    @staticmethod
+    def type_var_string(name, filename):
+        """ Generates code snipper to be added
+        name: name of variable
+        filename: where value of variable stored
+        """
+        output = "const char " + name + "[] =\n"
+        with open(filename) as f:
+            for line in f.read().splitlines():
+                output += "    \"" + line + "\\r\\n\"\n"
+            output = output[:-1] + ";\n"
+        return output
+
+    @staticmethod
+    def type_macro_bin(name, filename):
+        """ Generates code snipper to be added
+        name: name of macro
+        filename: where value of macro stored
+        """
+        output = "#define " + name + " {\n"
+        data = subprocess.check_output(["xxd", "-i", filename]).decode("utf-8")
+        output += '  ' + '  '.join(data.splitlines(keepends=True)[1:-2])
+        output = "".join([(line + (77-len(line))*" " + "\\\n") for line in output.splitlines()])
+        output += "}\n"
+        return output
+
+    @staticmethod
+    def type_macro_string(name, filename):
+        """ Generates code snipper to be added
+        name: name of macro
+        filename: where value of macro stored
+        """
+        output = "#define " + name + "\n"
+        with open(filename) as f:
+            for line in f.read().splitlines()[:-1]:
+                output += "    \"" + line + "\\r\\n\"\n"
+            output = output[:-1] + "\n"
+        output = "".join([(line + (75-len(line))*" " + "\\\n") for line in output.splitlines()])
```

Comment:
Why 75?

Reply:
No specific reason. Wide enough to contain stuffs